client/Lykke.Service.MarketMakerReports.Client/Models/InventorySnapshots/AssetBalance.cs

```diff
@@ -0,0 +1,16 @@
+using Newtonsoft.Json;
+
+namespace Lykke.Service.MarketMakerReports.Client.Models.InventorySnapshots
+{
+    public class AssetBalanceModel
+    {
+        [JsonIgnore]
+        public string Asset { get; set; }
```

Comment:
Why this property is ignored? This property contains asset id should it be called "AssetId"?

Reply:
It's ignored to avoid a lot of duplication in serialised json, as this type is in the array of balances which belongs to parent entity which already has a name of the asset. Actually, it can be completely removed from the model.